src/encoder.rs

```diff
@@ -2231,57 +2231,68 @@ fn write_tile_group_header(tile_start_and_end_present_flag: bool) ->
     buf.clone()
   }
 
-pub fn encode_frame<T: Pixel>(fi: &mut FrameInvariants<T>, fs: &mut FrameState<T>) -> Vec<u8> {
+pub fn encode_show_existing_frame<T: Pixel>(
```

Comment:
I would like a second opinion, but maybe `encode_existing_frame()` or something similar would be a better name. `encode_show_existing_frame()` makes sense, but is a bit unintuitive in my opinion. Maybe a short function comment would help.

Reply:
Naming things is hard :)

I'll try to put a reference to the specification since the concept itself is indeed not so intuitive.

Reply:
Good idea.